TodoApp/src/TodoApp.Api/ConnectionStrings.config

```diff
@@ -0,0 +1,3 @@
+﻿<connectionStrings>
+  <add name="DefaultConnection" connectionString="mongodb://admin:dummypassword12345@ds012198.mlab.com:12198/tododb" providerName="AWS"/>
```

Comment:
And now you need to clean up your history: https://help.github.com/articles/removing-sensitive-data-from-a-repository/